### PR TITLE
CDAP-15672. HTTP streaming source integration tests.

### PR DESCRIPTION
**Pull request by aonischuk**
Make already existing batch integration tests class an abstract class. And extend it by both HttpStreamingSourceETLTest and HttpBatchSourceETLTest.

### CODE REVIEW

**Comment by aonischuk** [comment]
@albertshau can you please review this?

Ps. 
code from the HttpSourceETLTest.java is mostly moved from HttpBatchSourceETLTest.java. Not a new code 90%.
So the actual changes are smaller here. Than looks.

File: src/test/java/io/cdap/plugin/http/etl/HttpSourceETLTest.java [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+import org.junit.rules.TestName;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+public abstract class HttpSourceETLTest extends HydratorTestBase {
```
Comment (albertshau) [inline src/test/java/io/cdap/plugin/http/etl/HttpSourceETLTest.java:39]: Can you comment on which part is different and should be reviewed?

Comment (aonischuk) [reply src/test/java/io/cdap/plugin/http/etl/HttpSourceETLTest.java:39]: I have separated the changes into two commits. So git recognizes what was moved. Now the review should be much easier.

Comment (aonischuk) [reply src/test/java/io/cdap/plugin/http/etl/HttpSourceETLTest.java:39]: So please only review the second commit. Instead of "Files changed" all together.

**Review by albertshau — APPROVED** [review]

**Pull request merged**